versionManager.js

```diff
@@ -10,31 +10,31 @@ var options = {
   method: 'GET'
 }
 
-var replaceStringInline = function(inlineRegex, replacement, paths) {
+var replaceStringInline = function (inlineRegex, replacement, paths) {
   replace({
     regex: inlineRegex,
     replacement: replacement,
     paths: paths,
     recursive: true,
-    silent: true,
+    silent: true
   })
 }
 
-var req = https.request(options, function(res) {
+var req = https.request(options, function (res) {
   var data = ''
   res.setEncoding('utf8')
-  
+
   res.on('data', (chunk) => {
-    data+=chunk
+    data += chunk
   })
-  
+
   res.on('end', () => {
-    var minimumCoreVersion = data.split("releases/tag/v")[1].substring(0, 5);
+    var minimumCoreVersion = data.split('releases/tag/v')[1].substring(0, 5)
     // Add latest version of core to console config
-    replaceStringInline(/\"minimumCoreVersion\"\:\ \"[0-9]{1,2}\.[0-9]{1,2}\.[0-9]{1,2}\"/, `"minimumCoreVersion": "${minimumCoreVersion}"`, ['./app/config/default.json'])
+    replaceStringInline(/\"minimumCoreVersion\"\:\ \"[0-9]{1,2}\.[0-9]{1,2}\.[0-9]{1,2}\"/, `"minimumCoreVersion": "${minimumCoreVersion}"`, ['./app/config/default.json']) // eslint-disable-line 
```

Comment:
@tumijacob Why is this line being disabled?

Reply:
@bausmeier Standard doesn't allow back slashes on regex.

Reply:
@tumijacob Then remove them?